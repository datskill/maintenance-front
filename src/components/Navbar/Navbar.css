.root {
  flex-grow: 1;
}
.menuButton {
  margin-right: 20px;
}
.title {
  flex-grow: 1;
}
.title-centred {
  margin: auto !important;
}
.image-gabarit {
  width: 35px;
  height: 35px;
}
