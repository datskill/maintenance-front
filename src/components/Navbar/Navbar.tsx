import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import './Navbar.css'

export default function ButtonAppBar() {
    return (
        <div className="root">
            <AppBar position="static">
                <Toolbar>
                    <Typography variant="h6" className="title-centred">OpenData Paris</Typography>
                </Toolbar>
            </AppBar>
        </div>
    );
}