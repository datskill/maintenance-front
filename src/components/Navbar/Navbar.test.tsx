import Navbar from "./Navbar";
import * as React from "react";
import * as renderer from "react-test-renderer";

it("App renders correctly", () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
