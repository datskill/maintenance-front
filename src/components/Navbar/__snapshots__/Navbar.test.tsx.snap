// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="root"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <img
        alt=""
        className="image-gabarit"
        src=""
      />
      <h6
        className="MuiTypography-root title-centred MuiTypography-h6"
      >
        OpenData Paris
      </h6>
    </div>
  </header>
</div>
`;
